definitions/features/upstream_repositories.rb

```diff
@@ -14,6 +14,36 @@ def repoids_and_urls
     repoids_and_urls
   end
 
+  VERSION_MAPPING = {
```

Comment:
Thought to have some version mappings, can be removed once we test the upgrades with scenarios

Reply:
Shall we mark it with a TODO? If it has the potential to be longer lived, it would be nice to put this in a stand-alone file for easier locating and updating.